README.md

```diff
@@ -33,10 +33,11 @@ $ brew install faas-cli
 
 The main commands supported by the CLI are:
 
-* `faas-cli new` - creates a new function via a template in the current directory
 * `faas-cli build` - builds Docker images from the supported language types
```

Comment:
Please restore the original ordering which had deliberate ordering of the workflow for using the CLI.

Reply:
done.